upt_macports/templates/perl.Portfile

```diff
@@ -10,11 +10,33 @@ revision            0
 {% endblock %}
 
 {% block versions %}
+{%- if pkg.upt_pkg.requirements.run or pkg.upt_pkg.requirements.test or pkg.upt_pkg.requirements.build %}
 if {${perl5.major} != ""} {
+    {%- if pkg.upt_pkg.requirements.build|length > 0 %}
+
+    depends_build-append \
+                    {% for i in pkg.upt_pkg.requirements.build|sort(attribute='name') %}
+                    port:p${perl5.major}-{{i.name|replace('::','-')|lower}} {% if loop.index != pkg.upt_pkg.requirements.build|length %}\{% endif %}
```

Comment:
can ```{% if loop.index != pkg.upt_pkg.requirements.build|length %}``` not just be ```{% if not loop.last %}```?

Reply:
You're right. It is also what we do in Python.Portfile iirc.